### PR TITLE
feat(dep-graph): add dark mode

### PR DESCRIPTION
**Pull request by Lcarv20**
## Current Behavior

Currently there is no possibility to use different themes.

## Expected Behavior

Users can toggle between dark and light themes, or select to dynamically follow the system setting.

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by philipjfulcher** [comment]
Please update your PR title to match our commit guidelines, something like "feat(dep-graph): add dark mode"

**Comment by philipjfulcher** [comment]
This is a good start and takes care of changing our React components to dark mode. I'll work with our designer to determine some colors that fit the Nrwl color palette. 

One more thing we need to figure out is passing the dark mode setting to the graph to be able to alter the colors there. The graph is powered by a library called [Cytoscape](https://js.cytoscape.org/) which uses a canvas to render the graph. So it won't accept our CSS rules to change colors, but it does accept a very similar syntax for styles. You can find the existing styles for nodes and edges in `dep-graph/client/src/app/styles-graph`. 

The style rules accept a function as a setter. So for example, right now the background color on `allNodes` is set using `    backgroundColor: NrwlPalette.white`. This could be changed to a function like `    backgroundColor: () => isDarkMode ? NrwlDarkPalette.white : NrwlPalette.white` so based on the current theme, it can switch between the color from two different palettes. You can probably use one of the existing function you have for selecting the correct class in React here. 

Let me know if that is a good enough starting off point.

**Comment by Lcarv20** [comment]
Hey @philipjfulcher, thanks for the hint. I did something like you mention in the previous comment, however the <Tippy /> instances do not update dynamically, and the nodes, edges and arrows will only update after you hover. I didn't quite yet figure a way to dynamically update those. Also, I added my own colour scheme but you can always update them easily, or if you have a pallet in mind just tell me and I can make those changes.

**Comment by philipjfulcher** [comment]
@Lcarv20 - I pushed up a commit that should refresh the colors on the graph immediately. I also rebased on master to get the latest changes in and ran a format, which I'm hoping should clean-up the e2e failures.

File: dep-graph/client/src/app/dark-theme-styles.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Styling presets for dark mode, defined by agregating styling classes
```
Comment (johannesschobel) [inline dep-graph/client/src/app/dark-theme-styles.tsx:1]:
```suggestion
// Styling presets for dark mode, defined by aggregating styling classes
```

**Review by johannesschobel — CHANGES_REQUESTED** [review]
found a few typos that should be corrected

**Pull request merged**